### PR TITLE
Testing out "Request Changes" comment

### PR DESCRIPTION
**Pull request by luu-jennifer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
-# Ingenious Thoughts
-### Developed using React, Firebase, Axios and Sass
+# Thinker 
+## previously known as:// Ingenious Thoughts
+### Simple application where users can share their thoughts to the World. 
+## Developed using React, Firebase, Sass and Axios with VSCode on Apple M1 Pro.
```
Comment (chaarz) [inline README.md:4]: Maybe add the link to your live site in your README.md

**Review by chaarz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (luu-jennifer) [reply README.md:4]: Thanks Char!!! So helpful :)

Comment (luu-jennifer) [reply README.md:4]: Thank you, I've add the link.

**Review by chaarz — APPROVED** [review]
Great work Jen !

**Pull request merged**